src/ray/raylet/local_object_manager.h

```diff
@@ -165,8 +166,8 @@ class LocalObjectManager {
   void SpillObjectsInternal(const std::vector<ObjectID> &objects_ids,
                             std::function<void(const ray::Status &)> callback);
 
-  /// Release an object that has been freed by its owner.
-  void ReleaseFreedObject(const ObjectID &object_id);
+  /// Remove the owner of an object that has been freed by its owner.
+  void ReleaseFreedObject(const ObjectID &object_id, const rpc::Address &owner_address);
```

Comment:
nit: maybe we need a better name other than `ReleaseFreedObject` as it might not release the object.

Reply:
I think we should still keep the "release object" functionalities to the function name? What about `RemoveOwnerAndReleaseIfNoOwner` (too much implementation detail?)?

Reply:
How about UnpinObject?